definitions/infra-awsrekognition/summary_metrics.yml

```diff
@@ -0,0 +1,26 @@
+providerAccountName:
+  title: AWS account
+  unit: STRING
+  tag:
+    key: providerAccountName
+region:
+  title: Region
+  unit: STRING
+  tag:
+    key: aws.awsRegion
+requests:
+  goldenMetric: requests
+  title: Total Request Count
```

Comment:
For consistency, please lowercase the word "Count".

Reply:
addressed